TESTS/netsocket/README.md

```diff
@@ -1068,527 +1088,513 @@ Test whether timeouts are obeyed in TCPSockets.
+1. Call `TLSSocket::send("12345", 5)`.
 
 **Expected result:**
 
-TLSSocket::send must return an error
+`TLSSocket::send` must return an error.
 
 ### TLSSOCKET_SEND_REPEAT
 
-**Description:** Run SOCKET_SEND_REPEAT for TLSSOCKET by using port number 2009.
+**Description:** Run `SOCKET_SEND_REPEATÂ` for TLSSOCKET by using port number 2009.
 
 ### TLSSOCKET_SEND_TIMEOUT
 
-**Description:** Run SOCKET_SEND_TIMEOUT for TLSSOCKET by using port number 2009.
+**Description:** Run `SOCKET_SEND_TIMEOUTÂ` for TLSSOCKET by using port number 2009.
 
 ### TLSSOCKET_SEND_UNCONNECTED
 
 **Description:** 
 
-Make a HTTP request to an unconnected socket.
+Make an HTTP request to an unconnected socket.
 
 **Preconditions:**
 
-1.  Network interface and stack are initialized.
-2.  Network connection is up.
+1. Network interface and stack are initialized.
+1. Network connection is up.
 
 **Test steps:**
 
-1.  Create TLSSocket.
-2.  Call `TLSSocket::open()`.
-3.  Call `TLSSocket::send("12345", 5)`.
-4.  Call `TLSSocket::connect("echo.mbedcloudtesting.com", 2007)`.
+1. Create TLSSocket.
+1. Call `TLSSocket::open()`.
+1. Call `TLSSocket::send("12345", 5)`.
+1. Call `TLSSocket::connect("echo.mbedcloudtesting.com", 2007)`.
 
 **Expected result:**
 
-TLSSocket::send must return an error
+`TLSSocket::send` must return an error.
 
 ### TLSSOCKET_ECHOTEST
 
-**Description:** Run SOCKET_ECHOTEST for TLSSOCKET by using port number 2007.
+**Description:** Run `SOCKET_ECHOTESTÂ` for TLSSOCKET by using port number 2007.
 
 ### TLSSOCKET_ECHOTEST_NONBLOCK
 
-**Description:** Run SOCKET_ECHOTEST_NONBLOCK for TLSSOCKET by using port number 2007.
+**Description:** Run `SOCKET_ECHOTEST_NONBLOCKÂ` for TLSSOCKET by using port number 2007.
 
 ### TLSSOCKET_ENDPOINT_CLOSE
 
-**Description:** Run SOCKET_ENDPOINT_CLOSE for TLSSOCKET by using port number 2013.
+**Description:** Run `SOCKET_ENDPOINT_CLOSE` for TLSSOCKET by using port number 2013.
 
 ### TLSSOCKET_NO_CERT
 
 **Description:**
 
-Verify that TLS Socket fails to connect without certificate.
+Verify TLS Socket fails to connect without a certificate.
 
 **Preconditions:**
 
-1.  Network interface and stack are initialized.
-2.  Network connection is up.
+1. Network interface and stack are initialized.
+1. Network connection is up.
 
 **Test steps:**
 
-1.  Create TLSSocket, without adding a default certificate.
-2.  Call `TLSSocket::open()`.
-3.  Call `TLSSocket::connect("echo.mbedcloudtesting.com", 2009)`.
+1. Create TLSSocket, without adding a default certificate.
+1. Call `TLSSocket::open()`.
+1. Call `TLSSocket::connect("echo.mbedcloudtesting.com", 2009)`.
 
 **Expected result:**
 
-TLSSocket::connect must return an error if the certificate is not present.
+`TLSSocket::connect` must return an error if the certificate is not present.
 
 ### TLSSOCKET_RECV_TIMEOUT
 
 **Description:**
 
-Run TCPSOCKET_RECV_TIMEOUT for TLSSOCKET by using port number 2007.
+Run `TCPSOCKET_RECV_TIMEOUTÂ` for TLSSOCKET by using port number 2007.
```

Comment:
It looks like there's some weird formatting going on. Do we know what this symbol should be, or should we delete it?

Reply:
Sorry but i don't know that.